.pre-commit-config.yaml

```diff
@@ -35,3 +35,7 @@ repos:
     rev: v4.3.21
     hooks:
     -   id: isort
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.4.0
+    hooks:
+    -   id: check-json
```

Comment:
Do we want this in the default pre-commit or just in the all config for CI? Is there no noticeable extra time to run this?

I think we should try to keep the default pre-commit lean and fast.

Reply:
It will add an additional container on initial startup, which takes a bit of extra time, but is cached afterward.

The check run in less than a second on my machine.

Reply:
Screen recording:

![2019-12-13 12 19 35](https://user-images.githubusercontent.com/195327/70796558-e27d6e80-1da2-11ea-910b-1935941ffe3f.gif)

Reply:
Ok.

Reply:
Additionally, in normal working, pre-commit runs only on changed files, the above check runs on ALL files.